Examples/ObjectiveC/SwitchStylesExample.m

```diff
@@ -0,0 +1,90 @@
+#import "SwitchStylesExample.h"
+@import Mapbox;
+
+NSString *const MBXExampleSwitchStyles = @"SwitchStylesExample";
+
+@interface SwitchStylesExample () <MGLMapViewDelegate, UIPickerViewDataSource>
+
+@property (nonatomic) MGLMapView *mapView;
+@property (nonatomic) UIPickerView *stylePicker;
+@property (nonatomic) UIButton *button;
+@property (nonatomic, readonly) NSArray<NSDictionary *> *styleURLs;
+
+@end
+
+@implementation SwitchStylesExample
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+    
+    _styleURLs = @[
+                    @{@"Mapbox Light":[MGLStyle lightStyleURL]},
+                    @{@"Mapbox Streets":[MGLStyle streetsStyleURL]},
+                    @{@"Mapbox Dark":[MGLStyle darkStyleURL]},
+                    @{@"Mapbox Outdoors": [MGLStyle outdoorsStyleURL]},
+                    @{@"Mapbox Satellite": [MGLStyle satelliteStyleURL]}
+                  ];
+
+    self.mapView = [[MGLMapView alloc] initWithFrame: CGRectMake(0, 0, self.view.bounds.size.width, self.view.bounds.size.height - 50)];
+    self.mapView.autoresizingMask = UIViewAutoresizingFlexibleWidth | UIViewAutoresizingFlexibleHeight;
+    self.mapView.styleURL = [MGLStyle lightStyleURL];
+    [self.mapView setCenterCoordinate:CLLocationCoordinate2DMake(38.89, -77.03)];
+    [self.view addSubview:_mapView];
```

Comment:
Nit: change this to `self.mapView`. `_mapView` is effectively the same as `self.mapView`, but it’s good to maintain consistency from line-to-line within the same method.